ocis-pkg/CHANGELOG.md

```diff
@@ -0,0 +1,318 @@
+
+   We added a middleware that dismantles the `x-access-token` from the request header and makes
+   it available in the context.
+
+   https://github.com/owncloud/ocis-pkg/pull/46
+
+
+* Enhancement - Add `ocis.id` and numeric id claims: [#50](https://github.com/owncloud/ocis-pkg/pull/50)
+
+   We added an `ocis.id` claim to the OIDC standard claims. It allows the idp to send a stable
+   identifier that can be exposed to the outside world (in contrast to sub, which might change
+   whens the IdP changes).
+
+   In addition we added `uidnumber` and `gidnumber` claims, which can be used by the IdP as well.
+   They will be used by storage providers that integrate with an existing LDAP server.
+
+   https://github.com/owncloud/ocis-pkg/pull/50
+
+# Changelog for [2.2.1] (2020-04-21)
+
+The following sections list the changes in ocis-pkg 2.2.1.
+
+[2.2.1]: https://github.com/owncloud/ocis-pkg/compare/v2.2.0...v2.2.1
+
+## Summary
+
+* Bugfix - Pass flags to micro service: [#44](https://github.com/owncloud/ocis-pkg/pull/44)
+
+## Details
+
+* Bugfix - Pass flags to micro service: [#44](https://github.com/owncloud/ocis-pkg/pull/44)
+
+   Passed flags to the micro service to be able to pass cli arguments while starting a grpc service.
+
+   https://github.com/owncloud/ocis-pkg/pull/44
+
+# Changelog for [2.2.0] (2020-03-30)
+
+The following sections list the changes in ocis-pkg 2.2.0.
+
+[2.2.0]: https://github.com/owncloud/ocis-pkg/compare/v2.1.0...v2.2.0
+
+## Summary
+
+* Change - Add header to cors handler: [#41](https://github.com/owncloud/ocis-pkg/issues/41)
+
+## Details
+
+* Change - Add header to cors handler: [#41](https://github.com/owncloud/ocis-pkg/issues/41)
+
+   The `x-requested-with` header was added to allow ajax requests.
+
+   https://github.com/owncloud/ocis-pkg/issues/41
+
+# Changelog for [2.1.0] (2020-03-16)
+
+The following sections list the changes in ocis-pkg 2.1.0.
+
+[2.1.0]: https://github.com/owncloud/ocis-pkg/compare/v2.0.2...v2.1.0
+
+## Summary
+
+* Enhancement - Tracing middleware: [#35](https://github.com/owncloud/ocis-pkg/pull/35/)
+
+## Details
+
+* Enhancement - Tracing middleware: [#35](https://github.com/owncloud/ocis-pkg/pull/35/)
+
+   A new tracing middleware has been added to unpack context propagation
+
+   https://github.com/owncloud/ocis-pkg/pull/35/
+
+# Changelog for [2.0.2] (2020-03-04)
+
+The following sections list the changes in ocis-pkg 2.0.2.
+
+[2.0.2]: https://github.com/owncloud/ocis-pkg/compare/v2.0.1...v2.0.2
+
+## Summary
+
+* Enhancement - Allow http services to register handlers: [#33](https://github.com/owncloud/ocis-pkg/pull/33)
+
+## Details
+
+* Enhancement - Allow http services to register handlers: [#33](https://github.com/owncloud/ocis-pkg/pull/33)
+
+   Added a handler option on http services
+
+   https://github.com/owncloud/ocis-pkg/pull/33
+
+# Changelog for [2.0.1] (2020-02-05)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)